doc/en/docs/docker.md

```diff
@@ -31,28 +31,25 @@ Dockerfile folder, e.g., tool/docker/devel/
 
 The `<TAG>` is named as
 
-    devel|runtime[-OS][-CUDA|OPENCL][-CUDNN]
+    devel|runtime[-OS][-CUDA|MKLDNN][-CUDNN]
 
 * devel: development images with all dependent libs' header files installed and SINGA's source code; runtime: the minimal images which can run SINGA programs.
-* OS: ubuntu, ubuntu14.04, centos, centos6
-* CUDA: cuda, cuda8.0, cuda7.0
-* CUDNN: cudnn, cudnn5, cudnn4
-* OPENCL: opencl, opencl1.2
+* OS: ubuntu18.04, ubuntu16.04, centos6
+* CUDA: cuda10.0, cuda9.0
+* CUDNN: cudnn7
+* MKLDNN: mkldnn0.18
 
-By default, if the version is not included in the tag, the latest stable version is used.
-The default OS is Ubuntu. The version is the latest stable version (e.g., 16.04 for now).
-For -cuda version, the **cudnn** is included by default. Their versions are also the latest stable version, i.e., cuda8.0 and cudnn5 for now.
+Here are some example tags:
 
-Here are some example tags,
+`devel-cuda9-cudnn7`, `devel-cuda9-cudnn7`, `devel-cuda10-cudnn7`, `devel-mkldnn0.18`, `devel-centos6`, `runtime`
```

Comment:
I think we can just provide devel-cuda9-cudnn7, devel-cuda10-cudnn7, devel-cpu, runtime-gpu (use cuda10-cudnn7, conda install) and runtime-cpu (conda install),.
mkldnn is enabled by default.

Reply:
Hi @nudles Thank you for the review. This is updated.